ranges/typedranges.nim

```diff
@@ -160,7 +160,10 @@ proc `[]=`*[T, U, V](r: MutRange[T], s: HSlice[U, V], v: openarray[T]) =
     raise newException(RangeError, "different lengths for slice assignment")
 
 template toOpenArray*[T](r: Range[T]): auto =
-  when (NimMajor,NimMinor,NimPatch)>=(0,19,9):
+  when false:
```

Comment:
Hrm, isn't the correct fix here just to change the version check:

```
when (NimMajor,NimMinor,NimPatch)>=(0,19,2): # minor changed from 9 to 2
```

This way, the code will work for people who are still using 0.19.0

Reply:
The code still works for 0.19.0 and 0.19.2. The part that I disabled used to work for Nim HEAD, but doesn't any more.

You can argue that it needs to be deleted completely, but not that it needs to be enabled at all, because there's no Nim version in which it works any more.

Reply:
thx for the ping, seems to work so no issues with this (apart from why isn't `UncheckedArray` allowed in this case ? @zah )